config/jobs/cert-manager-trust/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+- joshvanl
+reviewers:
+- joshvanl
+labels:
+- area/cert-manager-trust
```

Comment:
suggestion: I lose track of what prow actually cares about so I don't know how much `OWNERS` matters, but if we're adding it, should it at least contain the current members of the cert-manager-maintainers group as approvers and reviewers?

I note the reference to "milestone-maintainers" in `config/plugins.yaml` in this PR

Reply:
My understanding is that this will automatically ping me to review if a PR is opened to change anything in this dir.

Happy to change it but don’t think it matters too much… a testing PR is normally always followed up by a slack message.